<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClooneyScanner</class>
 <widget class="QMainWindow" name="ClooneyScanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clooney Scanner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="data_preview">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>440</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reject_button">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>440</y>
      <width>200</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string>Reject</string>
    </property>
   </widget>
   <widget class="QLabel" name="scan_preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>340</width>
      <height>440</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ForbiddenCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 100%;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
